src/main_app/db/services/settings_service.py

```
@@ -182,7 +112,39 @@ def create_setting(
+            title=title,
+            value_type=value_type,
+            value=str(value) if value is not None else None,
+        )
+        db.session.add(orm_obj)
+        try:
+            db.session.commit()
+            return True
+        except IntegrityError:
+            db.session.rollback()
+            return False
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)